plenum/server/replica.py

```diff
@@ -769,6 +770,7 @@ def create3PCBatch(self, ledger_id):
             self.txnRootHash(ledger_id)
         ]
 
+        # TODO: Should only happen on master replica
```

Comment:
I don't agree with it.
BLS is already stored for master Replica only. Validation and calculation is intentionally done for all replicas, because all replicas musty do the same amount of work, otherwise RBFT just doesn't work correctly.

Reply:
Difference between master and backups is minor (a function call) since backups don't have BLS sig or state root but ok, i agree to the principle. Removing